browser-test/src/question_lifecycle.test.ts

```diff
@@ -255,4 +255,36 @@ describe('normal question lifecycle', () => {
       ),
     ).toBeTruthy()
   })
+
+  it('redirects to draft question when trying to edit original question', async () => {
+    const {page} = await startSession()
+    page.setDefaultTimeout(4000)
```

Comment:
Is the timeout override necessary?

Reply:
Maybe not. Copied it from another test. Removed it and let's see if it becomes flaky.